README.md

```diff
@@ -51,8 +51,8 @@ Build Status
 [rhel6-build-badge]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7392/badge
 [rhel6-build]: https://devdiv.visualstudio.com/DevDiv/_build?_a=completed&definitionId=7392
 
-[alpine3.6-build-badge]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/8168/badge
-[alpine3.6-build]: https://devdiv.visualstudio.com/DevDiv/_build?_a=completed&definitionId=8168
+[linux-musl-build-badge]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/8168/badge
+[linux-musl-build]: https://devdiv.visualstudio.com/DevDiv/_build?_a=completed&definitionId=8168
```

Comment:
@weshaggard 
New VSO build def for Linux-musl: https://devdiv.visualstudio.com/DevDiv/Default/_build/index?context=allDefinitions&path=%5CDotNet%5CCLI%5Cmaster&definitionId=9036

Reply:
@johnbeisner I though we were just going to rename the old definition and update the build parameters?

Reply:
@weshaggard 
I re-thought how this change would roll-through the branches - in any scenario something needed to be updated. Since this change would probably not roll-back into servicing 2.1.xxx - therefore, just adding new branches to the 'Linux-musl' VSO build def triggers is best instead of conditioning the 'runtime-Id' on 'branch'.